settings.gradle

```diff
@@ -59,3 +59,4 @@ include 'data-prepper-plugins:date-processor'
 include 'data-prepper-expression'
 include 'data-prepper-plugins:mutate-string-processors'
 include 'data-prepper-plugins:s3-source'
+include 'core-peer-forwarder'
```

Comment:
You added a new project here. But, I don't see any project files under `core-peer-forwarder`. We can just remove this for now.